### DISCUSSION TITLE
phonecall model or nova-2?

### DISCUSSION THREAD

**Discussion by team-deepgram** [deepgram/community#594] [General help] [answered]
For a twilio integration, would accuracy be better using the phonecall model or nova-2?

> [Posted by mcqx2](https://discord.com/channels/1108042150941294664/1204920045021700207/1204920045021700207)

**Comment by lukeocodes** [comment, marked as answer]
Nova-2-phonecall is your best bet for phonecall accuracy. Failing that, Nova-2 may work. Depends on the quality of audio.
